test/plugin/file/csv/csvexporter.test.js

```diff
@@ -27,11 +27,6 @@ describe('plugin.file.csv.CSVExporter', function() {
     expect(result).toBeNull();
   });
 
-  it('does not process features lacking a geometry', function() {
-    var result = ex.processItem(new ol.Feature());
-    expect(result).toBeNull();
-  });
-
```

Comment:
This is not a good test fix. The better fix is to change the test from "It should not process features lacking a geometry" to "It should process features lacking a geometry" so that your new code path is tested.